### PR TITLE
Instrument result importing changes wrt senaite.instruments

### PR DESCRIPTION
**Pull request by mikejmets**
## Description of the issue/feature this PR addresses
With the introduction of senaite.instruments some changes are required in core to accommodate this.

Linked issue: 
https://github.com/senaite/senaite.core/issues/1341 Moved Agilent instruments from core to senaite.instruments

## Current behavior before PR
New instruments in senaite.instruments do not import correctly

## Desired behavior after PR is merged
New instruments in senaite.instruments import correctly

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: bika/lims/exportimport/instruments/instrument.pt [hunk 1]
```diff
@@ -4,6 +4,8 @@
 <label for='instrument_results_file_format'>Format</label>&nbsp;
 <select name="instrument_results_file_format" id="instrument_results_file_format">
     <option value='csv'>CSV</option>
+    <option value='xls'>XLS</option>
+    <option value='xlsx'>XLSX</option>
```
Comment (xispa) [inline bika/lims/exportimport/instruments/instrument.pt]: As far as I know, this is the "default" instrument import view, so these options might depend on the available parsers for a given instrument. Maybe one instrument has parsers for all three formats, but other instruments will only have parsers for just one (e.g. CSV). Or, are you handling all them generically?

File: bika/lims/content/abstractanalysis.py [hunk 2]
```diff
@@ -551,6 +551,9 @@ def calculateResult(self, override=False, cascade=False):
 
         # Calculate
         formula = calc.getMinifiedFormula()
+        if not formula:
+            return False
```
[the remark this answers is not in the document]
Comment (xispa) [reply bika/lims/content/abstractanalysis.py]: I suggest to create another PR to take care of https://github.com/senaite/senaite.core/issues/1339

File: bika/lims/exportimport/instruments/instrument.pt (hunk 1 above)
Comment (mikejmets) [reply bika/lims/exportimport/instruments/instrument.pt]: You are correct, I could define a specific a local view for relevant instruments.  Another question I have is what benefit does the Format field have at all? Are the some instruments that allow more that one format?

File: bika/lims/content/abstractanalysis.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mikejmets) [reply bika/lims/content/abstractanalysis.py]: Removed from code base and PR description

File: bika/lims/exportimport/instruments/instrument.pt (hunk 1 above)
Comment (xispa) [reply bika/lims/exportimport/instruments/instrument.pt]: Some instruments are capable to generate results files in different formats (e.g. XML + CSV), so the idea was the keep same form view for a given instrument, even if it's capable to generate results files in different formats.

Comment (mikejmets) [reply bika/lims/exportimport/instruments/instrument.pt]: Cool. I'll revert this and create local templates.

Comment (mikejmets) [reply bika/lims/exportimport/instruments/instrument.pt]: Removed commit from branch and updated PR

**Pull request merged**